pano.py

```diff
@@ -81,7 +81,10 @@ def run(self):
         pygame.camera.init()
         self.fuente = pygame.font.Font(None, 60)
         self.camlist = camera.list_cameras()
-        self.camera = camera.Camera(self.camlist[0], self.size, "RGB")
+        if self.camlist:
+            self.camera = camera.Camera(self.camlist[0], self.size, "RGB")
+        else:
+            print("No cameras detected.")
```

Comment:
A child isn't going to see this message.  So it is not useful.  What do you intend to happen if there is no camera?

Reply:
Then nothing can be done except for seeing camera going out of index, i dont think anything else can be done.

Reply:
I don' t believe nothing can be done.  Other activities that need a camera do various other things.  Think of the child using the software.  What should they be told, if they have a computer without a camera, and they run this activity?  How should they be told?